src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -33,10 +45,21 @@ def fdb_vlanmac(self, fdb):
                     vlan_id = vlan_id.decode()
                 self.bvid_vlan_map[fdb["bvid"]] = vlan_id
         else:
-            return None
+            return None, None
         if not isinstance(vlan_id, str):
-            return None
-        return (int(vlan_id),) + mac_decimals(fdb["mac"])
+            return None, None 
+        return (int(vlan_id),) + mac_decimals(fdb["mac"]), int(vlan_id)
+
+    def get_tp_fdb_status(self, ent, fdb):
+        #TODO:add code for self type
```

Comment:
>#TODO:add code for self type [](http://example.com/codeflow?start=8&length=28)

What is the plan on this TODO? possible to do in this PR?

Reply:
No plan